java/mdsplus-api/src/main/java/mds/data/descriptor_r/Function.java

```diff
@@ -30,13 +30,16 @@ public op_rec(final String symbol, final OPC opc, final byte prec, final byte lo
 			this.lorr = lorr;
 		}
 	}
-	private static String						newline;
+
+	public static final String LINEENDING_UNIX = "\n\t\t\t\t\t\t\t";
```

Comment:
What's with all the tabs?

Reply:
its about the linebreak \n vs \n\r the tabs are for the indent